### PR TITLE
Add initial cohort of Jupyter Distinguished Contributors

### PR DESCRIPTION
**Pull request by afshin**
Adds the Distinguished Contributors section above the SC.

Screen Shot 2020-10-16 at 18 28 20
Screen Shot 2020-10-16 at 18 10 47

### CODE REVIEW

**Review by Carreau — APPROVED** [review]
Ah, well, there was already a distinguish contributor section in sterign council html.

**Comment by dsblank** [comment]
Not sure why I don't have a picture... let me know what you need :)

File: about.md [hunk 1]
```diff
@@ -3,6 +3,95 @@ layout: page_md
...
+    gh_handle: gnestor
+    linkedin: https://linkedin.com/in/grantnestor/
+    twitter_handle: senornestor
+  - name: Ian Rose
+    photo: Ian_Rose.jpg
+    gh_handle: ian-r-rose
+    twitter_handle: IanRRose
+    personal: https://ian-r-rose.github.io/
+  - name: Douglas Blank
+    photo: placeholder.png
```
Comment (Carreau) [inline about.md:28]:
```suggestion
    photo: https://avatars.githubusercontent.com/u/168568
```

Comment (Carreau) [reply about.md:28]: nevermind, you use relative URL so will need to copy it.

Comment (dsblank) [reply about.md:28]: Ok, so you're set? Thanks!

Comment (afshin) [reply about.md:28]: We didn't want to use an avatar without permission @dsblank! I'm happy to update it now with your GH avatar!

Comment (dsblank) [reply about.md]: Perfect! Sorry if I missed a request!

**Review by choldgraf — APPROVED** [review]
<3

**Comment by dsblank** [comment]
If you merge this, I can show my Dad. Otherwise, he won't believe me :)

**Review by damianavila — APPROVED** [review]

**Comment by jasongrout** [comment]
Thanks everyone!

**Pull request merged**